### DISCUSSION TITLE
Progress bar in Multi Step Form

### DISCUSSION THREAD

**Discussion by kodecrash** [Budibase/budibase#6282] [Help] [answered]
How to enable progress bar in multi step form?

I want to achieve this user experience.

![image](https://user-images.githubusercontent.com/8296994/173228327-e8d697ac-0457-4c6b-9643-91fc4b19d771.png)

**Comment by melohagan** [comment, marked as answer]
Hi @kodecrash 

I have added a tutorial to the docs that covers this: https://docs.budibase.com/docs/form-steps#progress-bar

Hopefully that gives you an idea of what you can do!

**Reply by kodecrash** [reply]
Thanks for providing the tutorial. Is there any ready made template available which shows progress bar.

**Reply by melohagan** [reply]
Hey @kodecrash 
There's no official template, but feel free to explore this app export:
[form_steps-export.txt](https://github.com/Budibase/budibase/files/8898005/form_steps-export-1655195102497.txt)

**Reply by kodecrash** [reply]
Thank you @melohagan
